docs/Documentation/Drivers/Image/Params.md

```diff
@@ -1,145 +1,102 @@
+ds = xr.open_dataset(filename)
+ds.snow_band
 ```
 
-From the Tonic source code repository, Tonic can be installed using Python's `distutils`:
+# Parameter File Information
 
-```shell
-python setup.py install
-```
 
-This installs a top level script, `vic_utils`, into your `$CONDA_ROOT/envs/vic_test/bin` directory and the `tonic` package into your conda environment (e.g. `$CONDA_ROOT/lib/.../site-packages/tonic`). Note that for most `conda` installations, `$CONDA_ROOT` represents the directory where the root `conda` installation is (e.g. `~/anaconda`).
+The VIC model parameter file for the Image Driver, [Parameters File](Params.md), contains spatially distributed parameters describing the land surface.
 
-If you don't want to use the Anaconda installation I've shown above, you can build the package in your local python installation using:
-```shell
-python setup.py develop
-```
+# Soil Parameters
 
-Below are two methods for installing `tonic` if you do not use `anaconda`.  This may or may not require write permissions. Because this is not our preferred method for installing `tonic`, the user should read the [Python documentation](https://docs.python.org/3.5/install/) for further instructions on how to install Python packages.
+Soil parameter file serves three main purposes:
 
-### Option 2a:  Using a local Python Install (With Write Permissions)
+*   Define the cell ID number of each grid cell. This ID number is essentially a database key that links a grid cell to its parameters in the various parameter files.
+*   Define the grid cell soil parameters
+*   Define initial soil moisture conditions, to be used in the absence of an initial state file.
 
-If you have write permissions to the location of your Python distribution, you can just run
+The soil parameters are supplied to VIC in a NetCDF file, with a separate variable for each soil parameter.
 
-```shell
-python setup.py install
-```
+For a list of soil parameters, see the [Classic Driver Soil Parameter File](../Classic/SoilParam.md).
 
-from the top level Tonic directory.  This will install `tonic` into your `$PYTHONPATH`.
+To access individual soil parameters, for example, bulk density,  
+```Python
+import xarray as xr
+filename = '~workdir/example.params.vic5.nc'
+ds = xr.open_dataset(filename)
+ds.bulk_density
+```
 
-### Option 2b:  Using a local Python Install (Without Write Permissions)
+# Vegetation Parameters
 
-If you do not have write permissions, you can install Tonic in your local `$PYTHONPATH` by following these steps:
+For a list of vegetation file parameters, see the [Classic Driver Vegetation Parameter File](../Classic/VegParam.md).
 
-Create a `lib/python` directory in your `$HOME` directory:
+To access individual vegetation parameters, for example, gridcell number,
 
-```shell
-mkdir -p $HOME/lib/python/
+```Python
+import xarray as xr
+filename = '~workdir/example.params.vic5.nc'
+ds = xr.open_dataset(filename)
+ds.gridcell
```

Comment:
I think we can drop the python examples in lieu of tables of what parameters are needed, their units, shapes, and descriptions.

| Parameter | Units | Shape          | Description                           |
|-----------|-------|----------------|---------------------------------------|
| gridcell  | -     | ny, nx         | Grid cell number                      |
| depth     | m     | nlayer, ny, nx | Thickness of each soil moisture layer |